AnnoyancesFilter/Popups/sections/subscriptions_specific.txt

```diff
@@ -51,6 +51,7 @@ gaugau.futabanet.jp,newscientist.com,jiji.com,edp24.co.uk,technologyreview.com,n
 !
 ! SECTION: Subscriptions - Regular rules
 !
+design-ac.net##button:has-text(シェア)
```

Comment:
I would prefer something more specific, like:
```adblock
design-ac.net#?#.template-description-container > button.ant-btn:has(> span:contains(シェア))
```

It looks like a share button, so please move it to `! NOTE: Specific rules` section in https://github.com/AdguardTeam/AdguardFilters/blob/master/SocialFilter/sections/specific.txt